manual/osgi/README.md

```diff
@@ -70,6 +70,46 @@ CqlSession session = CqlSession.builder()
     .build();
 ```
 
+### Using a custom `ClassLoader` for Application Bundled configuration
+
+In addition to specifying a `ClassLoader` when constructing a `Session`, you can also specify
+a `ClassLoader` instance on certain `DriverConfigLoader` methods for cases when your OSGi
+application bundle provides overrides to Driver configuration defaults. This is typically done by
+including an `application.conf` file in your application bundle. For example, if you want to use
+[DriverConfigLoader.fromClasspath] to specify a different resource name (other than "application"):
```

Comment:
Actually it would be nicer if we started by explaining the general case: using the default config loader with a custom class loader:

```
CqlSession session = CqlSession.builder()
    .withClassLoader(classLoader)
    .withConfigLoader(new DefaultDriverConfigLoader(classLoader))
    .build();
```

Reply:
I pushed a small commit that propagates the session builder's CL to the config loader, iif the user doesn't specify their own CL. SO the above snippet can be shortened to:

```
CqlSession session = CqlSession.builder()
    .withClassLoader(classLoader)
    .withConfigLoader(new DefaultDriverConfigLoader(classLoader))
    .build();
```